polaris-react/src/components/Avatar/Avatar.tsx

```diff
@@ -142,6 +142,8 @@ export function Avatar({
   // Use `dominant-baseline: central` instead of `dy` when Edge supports it.
   const verticalOffset = '0.35em';
 
+  const squareFontSize = (initials?.length || 0)<=2 ? 15.5 : 12.5;
```

Comment:
This approach has three different font-sizes but here we only see two and there is also a third below. We should probably just set `fontSize` in one place and move the logic out of the below section.

I also think that this could live in CSS or have some other adjustments like reducing the `letter-spacing`. You might even be able to use `clamp` and some percentages which might be a better scalable solution.

Reply:
@clarkjennings thanks for moving it into one spot. What do you think about the other CSS approaches?